package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@pxblue/react-auth-shared",
     "description": "Re-usable react logic for Authentication and Registration within Eaton applications. For use with @pxblue/react-native-auth-workflow and @pxblue/react-auth-workflow.",
-    "version": "3.2.1",
+    "version": "3.3.0",
```

Comment:
if you make this a beta, it will auto publish a version that you can use in the workflow